openshift/fluent-bit.cm.yaml

```diff
@@ -56,8 +59,28 @@ objects:
           # add namespace
           Record  namespace ${NAMESPACE}
 
+        # append tag corresponding to log level
+        # and copy to a new 'ECS record', for pushing to AWS ElasticSearch service.
+        [FILTER]
+          Name          rewrite_tag
+          Match         app
+          Rule          $level ([a-zA-Z]*)$ $TAG.$level true
+          Emitter_Name  re_emitted
+
+        # modify ECS records to include more key/value pairs for output to AWS ElasticSearch
+        [FILTER]
+          Name    lua
+          Match   app.http
+          script  script.lua
+          call    ecsMap
+
+        # send http level ECS records (with tag: 'app.http') to AWS ElasticSearch
+        # [OUTPUT]
+        #   Name http
+        #   Match app.http
```

Comment:
We'll probably want this to be uncommented and then openshift templated so that it can handle sending network traffic to a specific endpoint, whatever the syntax is supposed to be.